karspexet/urls.py

```diff
@@ -1,13 +1,17 @@
+from django.conf import settings
 from django.conf.urls import include, url
+from django.conf.urls.static import static
 from django.contrib import admin
 from django.views.generic import TemplateView
 
 urlpatterns = [
     url(r"^admin/", admin.site.urls),
     url(r"^economy/", include("karspexet.economy.urls")),
     url(r"^ticket/", include("karspexet.ticket.urls")),
-
     url(r"^500/", TemplateView.as_view(template_name="500.html")),
-
     url(r"^", include("cms.urls")),
 ]
+
+urlpatterns = static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT, show_indexes=settings.DEBUG) + urlpatterns
+urlpatterns = static(settings.MEDIA_THUMBS_URL, document_root=settings.MEDIA_THUMBS_ROOT, show_indexes=settings.DEBUG) + urlpatterns
+urlpatterns = static(settings.MEDIA_FILER_URL, document_root=settings.MEDIA_FILER_ROOT, show_indexes=settings.DEBUG) + urlpatterns
```

Comment:
Visst skulle man kunna skriva detta så här också? :) (Kanske blir mindre läsbart... Jag vet inte, tyckte `+ urlpatterns` gömde sig lite längst bort till höger)
```suggestion
urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT, show_indexes=settings.DEBUG)
urlpatterns += static(settings.MEDIA_THUMBS_URL, document_root=settings.MEDIA_THUMBS_ROOT, show_indexes=settings.DEBUG)
urlpatterns += static(settings.MEDIA_FILER_URL, document_root=settings.MEDIA_FILER_ROOT, show_indexes=settings.DEBUG)
```

Reply:
Skillnaden är väl att om du lägger på `+ urlpatterns` på slutet så prependar du de här `static`-anropen, medan om du gör `+= static(...)` så _appendar_ du dem på slutet istället.

Reply:
Det låter som en rimlig anledning! Troligtvis viktigt om det finns överlapp mellan URL:erna :)

Reply:
Ett alternativ kan vara att lägga alla media-patterns i början och sedan använda `+=` som du nämner. Det finns mycket skumheter i bilduppladdningskoden ska jag säga. :)